apps/designer/app/designer/shared/image-manager/image.tsx

```diff
@@ -0,0 +1,73 @@
+import { useMemo } from "react";
+import { styled } from "@webstudio-is/design-system";
+import { Image as WebstudioImage, loaders } from "@webstudio-is/image";
+import env from "~/shared/env";
+import type { RenderableAsset } from "../assets";
+import brokenImage from "~/shared/images/broken-image-placeholder.svg";
+
+type ImageProps = {
+  renderableAsset: RenderableAsset;
```

Comment:
I guess you could call the property asset, since to the image its always a renderable asset and we have a type there for clarity

Reply:
asset.asset then makes me cry, bad property name